pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -171,6 +178,31 @@ func (o *MustGatherOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, arg
 	if len(o.DestDir) == 0 {
 		o.DestDir = fmt.Sprintf("must-gather.local.%06d", rand.Int63())
 	}
+	// TODO: this should be in Validate() method, but added here because of the call to o.completeImages() below
+	if o.AllImages {
+		errStr := fmt.Sprintf("and --all-images are mutually exclusive: please specify one or the other")
+		if len(o.Images) != 0 {
```

Comment:
nit: maybe switch looks nicer than `if`s

Reply:
But a `switch` can be useful when checking for different possibility of same variable. Here we are checking for different flags in each `if` statement. I don't think we can replace that with `switch` in this particular case.